README.md

```
@@ -27,44 +32,67 @@ target_link_libraries(<your-project>
 ./build/benchmark/mr-math-bench
 ```
 
 #### More advanced benchmark configuration
-To build benchmarks you have to generate CMake with `-D MR_MATH_ENABLE_BENCHMARK=ON`. \
-Note that this will not modify `CMAKE_BUILD_TYPE` or add any optimizations to compiler flags. \
-In order to add tested compiler flags please specify the preset. \
-**Note that specifying preset doesn't affect `CMAKE_BUILD_TYPE` either**.
-#### Available presets:
-- `MR_MATH_PRESET_OPTIMIZED` \
+If you are interested in specific results, you can disable particular benchmarks with the following options:
+- `MR_MATH_ENABLE_VEC_BENCH`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)